### DISCUSSION TITLE
Escrevendo artigos em Inglês

### DISCUSSION THREAD

**Discussion by devigor** [frontendbr/forum#1185] [Discussão]
Opa pessoal!!
Eu estou querendo escrever artigos em Inglês com o intuito de melhorar minha escrita na linguá e aperfeiçoar meu Inglês. Porém minha grande dúvida é como eu irei saber se o que eu escrevi está certo, gostaria de saber se existe alguma forma de corrigir meu texto de forma online?

Grato desde já!!

**Comment by Marlysson** [comment]
Tem essa ferramenta que acho fantástica:

https://www.grammarly.com/

**Comment by giovannism20** [comment]
Acho mais legal ainda a ideia de escrever essa extensão para português...

**Comment by alexanmtz** [comment]
@devigor estou com um projeto que trabalharemos com traduções e você passa por nossas revisões e aprende na prática, cadastre-se no https://gitpay.me

**Comment by felipeorlando** [comment]
@alexanmtz maneira a proposta do GitPay. Tem umas alternativas gringas que usufruem dessa ideia pagando tanto e U$, quando em criptomoedas. Precisando de ajuda pra tocar o barco, conte comigo, acho animal a ideia.

---

Respondendo a sua duvida, @devigor, acredito que a ideia seja escrever e sempre ter um link com um formulário pra receber sugestões e melhorias do texto. Aí tu joga pra comunidade e eles te ajudam. 😉

**Comment by alexanmtz** [comment]
Olá @felipeorlando , tem muitas iniciativas legais sim e estamos seguindo nesta mesma linha. Queremos trazer este tipo de iniciativa para as comunidades de desenvolvedores no Brasil, e não só no desenvolvimento como em documentação também, aí entra as colaborações com traduções como neste caso aqui:
https://github.com/worknenjoy/truppie/pull/40

Estamos amadurecendo a idéia ainda de suportar criptomoedas também 💸 

E ótima idéia para o @devigor, e se o conteúdo dele estiver no Github, daí alguém pode enviar um Pull Request com as melhorias. Ele pode fazer um site no Github pages e deixar links disponíveis para sugestão.

Precisamos de ajuda sim! Dê uma alô para tarefas@gitpay.me 📬
